app/src/main/java/org/aplatanao/billing/cockpit/control/Log.java

```diff
@@ -1,15 +1,25 @@
 package org.aplatanao.billing.cockpit.control;
 
-import javafx.scene.control.TextArea;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
 import org.springframework.stereotype.Component;
 
-import java.time.LocalDate;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 
 @Component
-public class Log extends TextArea {
+public class Log extends LogTable {
+
+    private final ObservableList<org.aplatanao.billing.cockpit.models.Log> data = FXCollections.observableArrayList();
 
     private void log(String level, String message) {
-        appendText(String.format("%s %s: %s\n", LocalDate.now(), level, message));
+
+        DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
```

Comment:
use one of the predefined ISO formats from the `java.time.format.DateTimeFormatter` or just the `toString()` of a `java.time.LocalDateTime` instance